### ISSUE TITLE
Request: Ability to apply xEdit script to selected refs in Referenced By window

### ISSUE THREAD

**Issue by fadingsignal** [TES5Edit/TES5Edit#856]
Humble request! The _Referenced By_ tab/window is a very powerful tool to be able to quickly see groupings of references, especially when they are scattered throughout a worldspace.

I often find myself in situations where being able to apply an xEdit script directly to these references from this view would be extremely helpful.  Specifically selected references, as pictured, not all of them.

Thank you for your consideration!

![applyscript](https://user-images.githubusercontent.com/13229928/90995481-f3f15a00-e570-11ea-92fc-d70908abc992.jpg)

**Comment by zilav** [comment]
You can handle references of a record from specific plugin with a bit of additional coding, in this case you will be applying script to XMarker record
```
function Process(e: IInterface): Integer;
var
  i: Integer;
  ref: IInterface;
begin
  e := MasterOrSelf(e);
  for i := 0 to Pred(ReferencedByCount(e)) do begin
    ref := ReferencedByIndex(e, i);
    if SameText( GetFileName(ref), 'SuperCoolPlugin.esp' ) then begin
      // do something with ref
    end;
  end;
end;
```

**Comment by zilav** [comment]
Oh and if you are going to redirect ref to another base object, then use reverse loop
```for i := Pred(ReferencedByCount(e)) downto 0 do```

**Comment by ElminsterAU** [comment]
He wants to be able to multi-select a subset of the records in the "referenced by" tab to apply the script to, not all of them.

**Comment by fadingsignal** [comment]
@zilav Yeah as @ElminsterAU said, I'd like to be able to select certain records in Referenced By and apply script only to what is selected.  However, your code is also very useful, so I appreciate it as well :)

**Comment by fireundubh** [comment]
@fadingsignal Randomly found this request while searching for something unrelated in Discord.

I implemented this feature in 96333bc223bf3b9848b3aba3c16495defcc7d4fe for v4.1.4.

**Comment by ElminsterAU** [comment]
The PR for this was merged and should show up in 4.1.3h

Thanks @fireundubh
